meson_get_version.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+# encoding=utf-8
+
+import os
+import subprocess
+import sys
+
+try:
+    sys.stdout.write(subprocess.check_output([
```

Comment:
This can be done without Python. See e.g. https://github.com/swaywm/swaylock/blob/master/meson.build#L53

Reply:
haha hello, didn't expect to see you here :)

your solution works to set the define for the C files, but not for the project version; I guess I can simplify the whole thing for now, and if someone decides later that we need the project verion to be accurate as well we can change this.